.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: Test
```

Comment:
Loving the additional tests

Reply:
Yep! This is just the beginning. I want to move the shell scripts into easily testable Go code as you suggested.

Notice that I kind of hacked the default Cobra command: I removed sub-commands so you can just run `drupal9ci` but you can certainly have multiple sub commands so you could do, for example `drupal9ci check` to check for requirements and, when ready, run `drupal9ci install` to run the installer. We could also do this in a follow up.

Reply:
At some point (sorry I am throwing out a lot of thoughts here), I would like to rename (again, I know) this repository to `drupalci` since we will be able to support multiple Drupal versions. Thoughts about this?

Reply:
100% on this.